### ISSUE TITLE
Issue with svg text stroke

### ISSUE THREAD

**Issue by andrewrukol** [rsms/inter#303]
**Describe the bug**
When using svg text element with `stroke` applied, the resulting text "border" goes inside the characters instead of outlining just the outer shape. That seems to be actual only for variable font.

**To Reproduce**
1. Create `svg` element
2. Create `text` element inside of svg with "4" text inside
3. Set `font-family` to be "Inter Var"
4. Set `stroke-width` and `stroke` color

You can see example here: https://codepen.io/Rukols/pen/GRoXzQZ

**Expected behavior**
There should not be stroke line inside of the character. Stroke should only go outside the character shape.

**Screenshots**
<img width="499" alt="Screenshot 2020-07-15 at 23 52 08" src="https://user-images.githubusercontent.com/17158026/87594839-3f088980-c6f6-11ea-8c69-8a8a543993dc.png">

<img width="478" alt="Screenshot 2020-07-15 at 23 53 01" src="https://user-images.githubusercontent.com/17158026/87594880-5778a400-c6f6-11ea-8ba1-96cb2ba5f334.png">

**Environment**
- OS: macOS 10.15
- App that renders the font: Chrome
- Version of font 3.13 (using https://rsms.me/inter/inter.css)

**Comment by rsms** [comment]
Related to #292

TL;DR: An issue with certain web browsers. Use static fonts instead of variable fonts.

[See this comment for a longer explanation](https://github.com/rsms/inter/issues/292#issuecomment-674993644)
